build/build-image/cross/Dockerfile

```diff
@@ -72,7 +72,7 @@ RUN mkdir $TMPDIR \
             github.com/jteeuwen/go-bindata/go-bindata
 
 # Download and symlink etcd. We need this for our integration tests.
-RUN export ETCD_VERSION=v2.2.1; \
+RUN export ETCD_VERSION=v3.0.10; \
```

Comment:
You mean for testing purposes? If so, updating this is not necessary, see e.g.:
https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L304

where we already have etcd3 enabled (some older version, but still).